conda-recipes/llvmdev/meta.yaml

```diff
@@ -91,6 +97,10 @@ test:
     - findstr __svml_sin4 out.ll                             # [win]
     - grep __svml_sin4 out.ll                                # [unix and x86_64]
 
+    # LLD tests
+    - ld.lld --version  # [linux]
```

Comment:
Think this can be `# [unix]`, OSX and BSD should work ok too?

Reply:
I think so. Fixed in e6e5f865f757282dead600fe6949bbc391e13e01.